packages/alchemy/src/middleware/client.ts

```diff
@@ -26,9 +27,12 @@ export type ClientWithAlchemyMethods = PublicErc4337Client & {
             entryPoint: Address;
             userOperation: UserOperationRequest;
             dummySignature: Hex;
-            feeOverride?: {
-              maxFeePerGas: Hex;
-              maxPriorityFeePerGas: Hex;
+            overrides?: {
```

Comment:
are all of the `override` fields required if the object exists? or is it possible to only override individual items?

Reply:
@moldy530 this is a draft. don't review yet. will give ping for asking review soon.